kickstarts/partials/main/disk_layout.ks.erb

```diff
@@ -8,5 +8,10 @@ part /var/www/miq_tmp --ondrive=vda                     --size=10240 --fstype=xf
 part pv.2             --ondrive=vda                     --size=10240 --grow
 
 volgroup vg_system    --pesize=4096 pv.1
-logvol /              --name=lv_os   --vgname=vg_system --size=10240 --fstype=xfs
-logvol swap           --name=lv_swap --vgname=vg_system --size=4096  --grow
+logvol /              --name=lv_os             --vgname=vg_system --size=4680 --fstype=xfs
+logvol /var           --name=lv_var            --vgname=vg_system --size=2048 --fstype=xfs
+logvol /var/log       --name=lv_var_log        --vgname=vg_system --size=1024 --fstype=xfs
+logvol /var/log/audit --name=lv_var_log_audit  --vgname=vg_system --size=512  --fstype=xfs
+logvol /tmp           --name=lv_tmp            --vgname=vg_system --size=1024 --fstype=xfs
```

Comment:
I was under the impression we use `/var/www/miq_tmp` if it exists otherwise we use the temp/Tempfile location.  If so, I think we're ok here since we still have the existing ~10 GB temp disk mounted at that location.  I haven't looked at that code recently though.